infra/examples-dev/aws-msft-365/variables.tf

```diff
@@ -82,7 +82,6 @@ variable "enabled_connectors" {
 
   default = [
     "azure-ad",
-    "dropbox-business",
```

Comment:
Can we keep it ?

Reply:
eh, i don't think it makes a ton of sense by default, given how rare usage is